dm/ctl/common/config.go

```diff
@@ -117,7 +130,16 @@ func (c *Config) configFromFile(path string) error {
 }
 
 // adjust adjusts configs
-func (c *Config) adjust() {
+func (c *Config) adjust() error {
+	if c.RPCTimeoutStr == "" {
+		c.RPCTimeoutStr = defaultRPCTimeout
+	}
+	timeout, err := time.ParseDuration(c.RPCTimeoutStr)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	c.RPCTimeout = timeout
```

Comment:
should we check whether `timeout == 0`？

Reply:
added check.
1. if upgrade dm-master from old version, and use old config, `RPCTimeoutStr` is empty value and we force to use `defaultRPCTimeout`
2. if user set `RPCTimeoutStr` to a positive value, return error directly